### ISSUE TITLE
shape_predictor_5_face_landmarks.dat

### ISSUE THREAD

**Issue by axmadjon** [axmadjon/ComputerVision#2]

**Comment by pradeeshnarayan** [comment]
When i try to run this in android I am getting an error
"terminating with uncaught exception of type dlib::serialization_error: Unable to open storage/emulated/0/Download/shape_predictor_5_face_landmarks.dat for reading."

Should i download this file and paste it in some location?

**Comment by axmadjon** [comment]
@pradeeshnarayan 
yes you must upload the file to the /Download folder file [shape_predictor_5_face_landmarks.dat](http://dlib.net/files/shape_predictor_5_face_landmarks.dat.bz2)
and you must add a file [dlib_face_recognition_resnet_model_v1.dat](http://dlib.net/files/dlib_face_recognition_resnet_model_v1.dat.bz2)

or you can change the path [here](https://github.com/axmadjon/ComputerVision/blob/master/FaceTrack/app/src/main/java/uz/greenwhite/facetrack/FaceApp.java#L11), [and here](https://github.com/axmadjon/ComputerVision/blob/master/FaceTrack/app/src/main/java/uz/greenwhite/facetrack/FaceApp.java#L12)
